website/static/js/pages/base-page.js

```diff
@@ -73,7 +73,7 @@ $(document).on('click', '.project-toggle', function() {
     return false;
 });
 
-var NO_FOOTER_PATHS = ['/login/', '/getting-started/', '/register/'];
+var NO_FOOTER_PATHS = ['/', '/login/', '/getting-started/', '/register/', '/forgotpassword/'];
```

Comment:
I'm not sure we want to exclude the index page. @saradbowman @AndrewSallans Do we want to show the footer slidein on the home page?

Also, you will need to add `/account/`, as that also routes to the sign-in page.